app/views/partials/admin_tag_metadata.html

```diff
@@ -0,0 +1,20 @@
+<div class="container">
```

Comment:
maybe we should just name the file admin_tag.html


[image: --]
Friedger Müffke
[image: http://]about.me/friedger
<http://about.me/friedger>


2015-09-14 12:45 GMT+02:00 Said Tahsin Dane <notifications@github.com>:

> In app/views/partials/admin_tag_metadata.html
> <https://github.com/gdg-x/hub/pull/53#discussion_r39380795>:
>
> > @@ -0,0 +1,20 @@
> > +<div class="container">
>
> Oh I see. It is not a metadata file. It is the template for tag metadata.
> :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/hub/pull/53/files#r39380795>.
>